llvm/test/CodeGen/AMDGPU/fptrunc.ll

```diff
@@ -111,34 +111,36 @@ define amdgpu_kernel void @fptrunc_f64_to_f16(ptr addrspace(1) %out, double %in)
 ; SI-NEXT:    s_bfe_u32 s0, s7, 0xb0014
```

Comment:
I could be wrong but just looking at diff chunk changes in this test file, this test seem to regress (i.e., more lines added than removed)